guidedmodules/views.py

```diff
@@ -644,12 +650,18 @@ def render_markdown_field(field, output_format, **kwargs):
             if mq.spec.get("protocol"):
                 can_start_any_apps = True
 
-        if questions.get(mq.id) and 'task' in questions[mq.id]:
-            # print("\nquestions[mq.id]['task'].id", dir(questions[mq.id]['task']))
+        # if questions.get(mq.id) and 'task' in questions[mq.id]:
+        # print("** questions.get(mq.id)", questions.get(mq.id)) # Debug
+        if 'task' in questions[mq.id]:
```

Comment:
Why this change?  Will `questions.get(mq.id)` always exist?

Reply:
Line 653 and 654 were debug lines. I know this because if you look at Line 908 you will see the code repeated. `questions.get(mq.id)` will always reply with an id value or None and if None, the else condition is executed.

Reply:
if `questions[mq.id]` is None, won't you get a `TypeError: argument of type 'NoneType' is not iterable`?

Reply:
I answered your previous question poorly, and have edited my answer.

`questions.get(mq.id)` was part of a debug effort. `mq.id` is always defined within this code block.